docs/concepts/configuration/pod-priority-preemption.md

```diff
@@ -57,6 +57,27 @@ scheduler. After the feature is disabled, the existing Pods keep their priority
 fields, but preemption is disabled, and priority fields are ignored, and you
 cannot set `priorityClassName` in new Pods.
 
+
+## How to disable preemption
+
+In Kubernetes 1.11 and later, preemption is controlled by a kube-scheduler flag `disablePreemption`, which is set to `false` by default.
+
+If you want to disable preemption, just set `disablePreemption` to `true`, here's a sample configuration:
```

Comment:
s/just set `disablePreemption` to `true`,/just set `disablePreemption` to `true`. This will keep pod priority enabled while preemption is disabled. Here is a sample configuration:/

Reply:
Done!